src/types/page.urls.ts

```diff
@@ -1,4 +1,11 @@
+import { Templates } from "./template.paths";
+
+const SEPARATOR = "/";
+
 export const ACCESSIBILITY_STATEMENT = "/accessibility-statement";
+export const CONFIRM_COMPANY = SEPARATOR + Templates.CONFIRM_COMPANY;
```

Comment:
you could do the same with the accessibility-statement, change it to use the template name as the page name is the same as template.